sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveCatalogs.scala

```diff
@@ -194,18 +185,6 @@ class ResolveCatalogs(val catalogManager: CatalogManager)
         if !isSessionCatalog(catalog) =>
       CreateNamespace(catalog.asNamespaceCatalog, ns, c.ifNotExists, c.properties)
 
-    case DropNamespaceStatement(NonSessionCatalogAndNamespace(catalog, ns), ifExists, cascade) =>
-      DropNamespace(catalog, ns, ifExists, cascade)
-
-    case DescribeNamespaceStatement(NonSessionCatalogAndNamespace(catalog, ns), extended) =>
-      DescribeNamespace(catalog, ns, extended)
-
-    case ShowNamespacesStatement(NonSessionCatalogAndNamespace(catalog, ns), pattern) =>
-      ShowNamespaces(catalog, ns, pattern)
```

Comment:
We missed the fallback to the v1 `ShowDatabasesCommand`.

Reply:
Let me check, thanks.

Reply:
@cloud-fan could you clarify on the fallback?

"show databases" and "show namespaces" were merged by #26006 and both are handled by v2 `ShowNamespacesExec`.

Reply:
Ah I misremember that. So we forgot to add a migration guide for the schema change... (databaseName -> namespace) Anyway it's too late now, let's catch schema changes in the following development work.